lib/accept.mli

```diff
@@ -43,13 +56,14 @@ type encoding =
   | Identity
   | AnyEncoding with sexp
 
+(** Language tags, as a list of subtags.
+    ["en-gb"] is represented as [Language ["en"; "gb"]].
+    @see <http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.4> The spec.
+*)
 type language = Accept_types.language =
```

Comment:
I know about BCP47, I was also considering doing such a module, happy to see you did it already.

Thanks for the link. It's updated. Is there a guideline in mirage libraries on how to reference RFCs and so on ? I guess it's waiting for ppx and [reqtrace](https://github.com/infidel/reqtrace) ?